### PR TITLE
Model backend for group invites.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

File: mozillians/groups/admin.py [hunk 1]
```diff
@@ -179,7 +180,8 @@ class Meta:
         model = Group
         fields = '__all__'
         widgets = {
-            'curators': autocomplete.ModelSelect2Multiple(url='users:vouched-autocomplete')
+            'curators': autocomplete.ModelSelect2Multiple(url='users:vouched-autocomplete'),
+            'invites': autocomplete.ModelSelect2Multiple(url='users:vouched-autocomplete'),
```
Comment (johngian) [inline mozillians/groups/admin.py]: I am not sure if this line here is valid.

**Comment by johngian** [comment]
Please squash the migration to a single one.

File: mozillians/groups/models.py [hunk 2]
```diff
@@ -333,20 +335,16 @@ def user_can_leave(self, userprofile):
...
-                                on_delete=models.SET_NULL,
-                                related_name='group_invites')
-    redeemer = models.ForeignKey('users.UserProfile',
-                                 verbose_name=_lazy(u'Redeemer'),
-                                 related_name='group_invites_redeemed')
-    group = models.ForeignKey('Group', related_name='invites')
-    accepted = models.NullBooleanField()
+    inviter = models.ForeignKey('users.UserProfile', null=True, on_delete=models.SET_NULL,
+                                related_name='invite_sent')
+    redeemer = models.ForeignKey('users.UserProfile', related_name='group_invited')
```
Comment (johngian) [inline mozillians/groups/models.py]: Why are you removing the verbose name here?

Comment (johngian) [inline mozillians/groups/models.py]: I prefer the previous ``related_name``.

File: mozillians/groups/models.py [hunk 3]
```diff
@@ -333,20 +335,16 @@ def user_can_leave(self, userprofile):
...
-    inviter = models.ForeignKey('users.UserProfile',
-                                null=True, verbose_name=_lazy(u'Inviter'),
-                                on_delete=models.SET_NULL,
-                                related_name='group_invites')
-    redeemer = models.ForeignKey('users.UserProfile',
-                                 verbose_name=_lazy(u'Redeemer'),
-                                 related_name='group_invites_redeemed')
-    group = models.ForeignKey('Group', related_name='invites')
-    accepted = models.NullBooleanField()
+    inviter = models.ForeignKey('users.UserProfile', null=True, on_delete=models.SET_NULL,
```
Comment (johngian) [inline mozillians/groups/models.py]: Why are you removing the verbose name here?

File: mozillians/groups/models.py [hunk 4]
```diff
@@ -333,20 +335,16 @@ def user_can_leave(self, userprofile):
...
+                                related_name='invite_sent')
+    redeemer = models.ForeignKey('users.UserProfile', related_name='group_invited')
+    group = models.ForeignKey('Group')
+    accepted = models.BooleanField(default=False)
```
Comment (johngian) [inline mozillians/groups/models.py]: Why are you changing ``accepted``. By allowing the ``NULL`` value you can more explicitly see if an Invite is neither accepted nor rejected.

Comment (akatsoulas) [inline mozillians/groups/models.py]: This is a valid point - I will amend this.

File: mozillians/groups/models.py (hunk 2 above)
Comment (akatsoulas) [inline mozillians/groups/models.py]: This `related_name` returns the group that the user was invited to, so the previous name does not make much sense. Regarding the verbose name I could amend it

File: mozillians/groups/admin.py (hunk 1 above)
Comment (akatsoulas) [inline mozillians/groups/admin.py]: :+1:

**Comment by johngian** [comment]
Code looks OK. r+

**Pull request merged**